src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -610,3 +610,8 @@ export function fetchPatients(samples:Sample[]) {
         patientFilter
     });
 }
+
+export function getProfilesWithoutMutationProfile(molecularprofiles: MolecularProfile[]): MolecularProfile[] {
```

Comment:
@kalletlak maybe more clear if the name is something like:

`excludeMutationAndSVProfiles`

`FUSION` is a subset of `STRUCTURAL_VARIANT` and will be deprecated in the future anyway